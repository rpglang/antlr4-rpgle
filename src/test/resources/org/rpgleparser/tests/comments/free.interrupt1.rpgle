        IF X = 1
      * Interrupting comment
          AND Y=2;
        ENDIF;        