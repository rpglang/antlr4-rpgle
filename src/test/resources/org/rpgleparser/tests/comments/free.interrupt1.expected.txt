/*===TOKENS===*/
OP_IF                    <IF>
ID                       <X>
'='                      <=>
NUMBER                   <1>
HIDDEN:COMMENT_SPEC_FIXED< *>
HIDDEN:COMMENTS_TEXT     <Interrupting comment>
AND                      <AND>
ID                       <Y>
'='                      <=>
NUMBER                   <2>
FREE_SEMI                <;>
OP_ENDIF                 <ENDIF>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (block
      (ifstatement
        (beginif
          (op_if
            IF
            (expression
              (expression
                (expression (identifier (free_identifier (idOrKeyword X))))
                =
                (expression (number 1))
              )
              AND
              (expression
                (expression (identifier (free_identifier (idOrKeyword Y))))
                =
                (expression (number 2))
              )
            )
          )
          ;
        )
        (endif (op_endif ENDIF) ;)
      )
    )
  )
)
/*======*/